### PR TITLE
Update XCM-Transactor and XCM-Fees page

### PR DESCRIPTION
**Pull request by albertov19**
### Description

This PR updates the XCM-Fees page and the XCM-Transactor page to accomodate the `transactThroughSigned` method. More changes will be needed with the new proxy method being implemented (cc @hyd628 for translations, you might want to wait?)

I'll address the checklist and add the missing images later

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/xcm/fees.md [hunk 1]
```diff
@@ -44,33 +44,33 @@ An XCM message is comprised of a series of XCM instructions. As a result, differ
...
+The following sections will break down how to calculate XCM fees for Polkadot and Kusama. It's important to note that Kusama, in particular, uses benchmarked data to determine the total weight costs for XCM instructions and that some XCM instructions might include database reads/writes, which add weight to the call.
 
 There are two databases available in Polkadot and Kusama, RocksDB (which is the default) and ParityDB, both of which have their own associated weight costs for each network. 
 
 ### Polkadot {: #polkadot }
 
-As previously mentioned, Polkadot currently uses a [fixed amount of weight](https://github.com/paritytech/polkadot/blob/e76cd144f9dad8c1304fd1476f92495bbb9ad22e/runtime/polkadot/src/xcm_config.rs#L95){target=_blank} for all XCM instructions, which is `{{ networks.polkadot.xcm_instruction.weight.display }}` weight units.
+As previously mentioned, Polkadot currently uses a [fixed amount of weight](https://github.com/paritytech/polkadot/blob/e76cd144f9dad8c1304fd1476f92495bbb9ad22e/runtime/polkadot/src/xcm_config.rs#L95){target=_blank} for all XCM instructions, which is `{{ networks.polkadot.xcm_instruction.weight.transfer }}` weight units.
```
Comment (eshaben) [inline builders/xcm/fees.md:73]: this shouldn't be changed, there is no corresponding variable for `weight.transfer`

File: builders/xcm/fees.md [hunk 2]
```diff
@@ -79,35 +79,43 @@ Although Polkadot doesn't currently use database weight units to calculate costs
...
-Planck-DOT-Weight =  PlanckDOT-Mapped * (1 / ExtrinsicBaseWeight)
+XCM-DOT-COST = XCMInstrWeight * DOTWeightToFeeCoefficient
 ```
 
-Where the constant is calculated as follows:
+Where `DOTWeightToFeeCoefficient` is a constant (map to 1 cent), and can be calculated as:
 
 ```
-Planck-DOT-Weight = (10^10 / 10000) * (1 / {{ networks.polkadot.extrinsic_base_weight.numbers_only }})
+KSMWeightToFeeCoefficient = ( 10^10 / ( 10 * 100 )) * ( 1 / DOTExtrinsicBaseWeight )
```
Comment (eshaben) [inline builders/xcm/fees.md:95]:
```suggestion
DOTWeightToFeeCoefficient = ( 10^10 / ( 10 * 100 )) * ( 1 / DOTExtrinsicBaseWeight )
```

File: builders/xcm/fees.md [hunk 3]
```diff
@@ -79,35 +79,43 @@ Although Polkadot doesn't currently use database weight units to calculate costs
...
 ```
 
-As a result, `Planck-DOT-Weight` is equal to `{{ networks.polkadot.xcm_instruction.planck_dot_weight }} Planck-DOT`. Now, you can begin to calculate the final fee in DOT, using `Planck-DOT-Weight` as a constant, and `TotalWeight` as the variable:
+Using the actual values:
 
 ```
-Total-Planck-DOT = TotalWeight * Planck-DOT-Weight
-DOT = Total-Planck-DOT / DOTDecimalConversion
+DOTWeightToFeeCoefficient = (10^10 / ( 10 * 100 * {{ networks.polkadot.extrinsic_base_weight.numbers_only }})
```
Comment (eshaben) [inline builders/xcm/fees.md:101]:
```suggestion
DOTWeightToFeeCoefficient = ( 10^10 / ( 10 * 100 * {{ networks.polkadot.extrinsic_base_weight.numbers_only }} )
```

File: builders/xcm/fees.md [hunk 4]
```diff
@@ -142,33 +150,35 @@ On Kusama, the benchmarked base weights are broken up into two categories: fungi
...
-Planck-KSM-Weight =  PlanckKSM-Mapped * (1 / ExtrinsicBaseWeight)
+XCM-KSM-COST = XCMInstrWeight * KSMWeightToFeeCoefficient
 ```
 
-Where `Planck-KSM-Weight` is calculated as follows:
+Where `KSMWeightToFeeCoefficient` is a constant (map to 1 cent), and can be calculated as:
 
 ```
-Planck-KSM-Weight = (10^12 / 30000 * 10) * (1 / {{ networks.kusama.extrinsic_base_weight.numbers_only }})
+KSMWeightToFeeCoefficient = ( 10^12 / ( 10 * 30000 )) * ( 1 / KSMExtrinsicBaseWeight )
```
Comment (eshaben) [inline builders/xcm/fees.md:164]: I would do the same thing as you did in the DOT section here with the "Using the actual values:" and then showing the actual values

File: builders/xcm/fees.md [hunk 5]
```diff
@@ -180,52 +190,58 @@ As an example, you can calculate the total cost of KSMs for sending an XCM messa
...
-    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-    | [{{ networks.moonriver.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonriver/src/xcm_config.rs#L208){target=_blank} | [{{ networks.moonriver.xcm.instructions.wei_per_weight.display }}](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L128){target=_blank} |
+1. Fees paid in the reserve token (native tokens like GLMR, MOVR, or DEV)
+2. Fees paid in external assets (XC-20s)
```
Comment (eshaben) [inline builders/xcm/fees.md:201]: Not sure what you think, but I feel like we typically only use numbers for steps and bullet points for other lists... either way not really a big deal

File: builders/xcm/fees.md [hunk 6]
```diff
@@ -180,52 +190,58 @@ As an example, you can calculate the total cost of KSMs for sending an XCM messa
...
-    |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
-    | [{{ networks.moonbeam.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbeam/src/xcm_config.rs#L201){target=_blank} | [{{ networks.moonbeam.xcm.instructions.wei_per_weight.display }}](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L128){target=_blank} |
+The following sections will break down how to calculate XCM fees for Moonbeam-based networks. There are two main scenarios:
```
Comment (eshaben) [inline builders/xcm/fees.md:198]: I think this makes more sense after the table 🤷‍♀️

File: builders/xcm/fees.md [hunk 7]
```diff
@@ -180,52 +190,58 @@ As an example, you can calculate the total cost of KSMs for sending an XCM messa
...
+| [{{ networks.moonbeam.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbeam/src/xcm_config.rs#L201){target=_blank} | [{{ networks.moonriver.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonriver/src/xcm_config.rs#L208){target=_blank} | [{{ networks.moonbase.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbase/src/xcm_config.rs#L219){target=_blank} |
+
+
+### Fee Calculation for Reserve Assets {: #moonbeam-reserve-assets }
+
+For each XCM instruction, the weight units are converted to balance units as part of the fee calculation. The amount of weight and Wei per weight for each of the Moonbeam-based networks is as follows:
```
Comment (eshaben) [inline builders/xcm/fees.md:212]:
```suggestion
For each XCM instruction, the weight units are converted to balance units as part of the fee calculation. The amount of Wei per weight for each of the Moonbeam-based networks is as follows:
```

File: builders/xcm/fees.md [hunk 8]
```diff
@@ -79,35 +79,43 @@ Although Polkadot doesn't currently use database weight units to calculate costs
...
 ```
-Total-Planck-DOT = TotalWeight * Planck-DOT-Weight
-DOT = Total-Planck-DOT / DOTDecimalConversion
+DOTWeightToFeeCoefficient = (10^10 / ( 10 * 100 * {{ networks.polkadot.extrinsic_base_weight.numbers_only }})
+```
+
+As a result, `DOTWeightToFeeCoefficient` is equal to `{{ networks.polkadot.xcm_instruction.planck_dot_weight }} Planck-DOT`. Now, you can begin to calculate the final fee in DOT, using `DOTWeightToFeeCoefficient` as a constant and `TotalWeight` as the variable:
+
+```
+XCM-Planck-DOT-COST = TotalWeight * DOTWeightToFeeCoefficient
```
Comment (eshaben) [inline builders/xcm/fees.md:107]: shouldn't `Planck` just be capitalized too? Either that or `COST` should be lower case?

File: builders/xcm/fees.md [hunk 9]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
 Then the calculation for the transfer is:
 
 ```
-Cost = ({{ networks.moonbeam.xcm.units_per_second.xcdot.numbers_only }} / 10^10) * {{ networks.moonbeam.xcm.message.transfer.exec_time }}
+XCM-Cost = ({{ networks.moonbeam.xcm.units_per_second.xcdot.transfer_numbers_only }} / 10^10) * {{ networks.moonbeam.xcm.message.transfer.exec_time }}
+```
+
+The total cost to transfer Alice's DOT to Alith's account for xcDOT is `{{ networks.moonbeam.xcm.message.transfer.xcdot_cost }} xcDOT`.
+
+## XCM-Transactor Fees
```
Comment (eshaben) [inline builders/xcm/fees.md:296]:
```suggestion
## XCM-Transactor Fees {: #xcm-transactor-fees }
```

File: builders/xcm/fees.md [hunk 10]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+The [XCM-transactor pallet](/builders/xcm/xcm-transactor/) builds an XCM message to remotely transact in other chains of the ecosystem.
```
Comment (eshaben) [inline builders/xcm/fees.md:298]:
```suggestion
The [XCM-transactor pallet](/builders/xcm/xcm-transactor/){target=_blank} builds an XCM message to remotely transact in other chains of the ecosystem. 
```

File: builders/xcm/fees.md [hunk 11]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+The [XCM-transactor pallet](/builders/xcm/xcm-transactor/) builds an XCM message to remotely transact in other chains of the ecosystem. 
+
+There are two different ways developers can remotely transact through the pallet:
+
+1. [`transactThroughDerivative`](/builders/xcm/xcm-transactor/#xcmtransactor-transact-through-derivative){target=_blank}
+2. [`transactThroughSigned`](/builders/xcm/xcm-transactor/#xcmtransactor-transact-through-signed){target=_blank}, in which the dispatcher account in the destination chain is a multilocation-derived account and must have enough funds to cover XCM execution fees plus whatever call is being remotely executed
+
+Generally speaking, the XCM instructions normally involved for remote execution are:
+
+ - Instructions to handle tokens in the origin chain. This can be either moving tokens to a sovereign account or burning the corresponding [XC-20](/builders/xcm/xc20/overview/){target=_blank} so that it can be used in the target chain. These instructions get executed in the origin chain
```
Comment (eshaben) [inline builders/xcm/fees.md:307]: is this meant to be here? seems like it was just supposed to be a list of the actual instructions

File: builders/xcm/fees.md [hunk 12]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+ - [`WithdrawAsset`](https://github.com/paritytech/xcm-format#withdrawasset){target=_blank} - gets executed in the target chain. Removes assets and places them into the holding register
+ - [`BuyExecution`](https://github.com/paritytech/xcm-format#buyexecution){target=_blank} - gets executed in the target chain. Takes the assets from holding to pay for execution fees. The fees to pay are determined by the target chain
+ - [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} - gets executed in the target chain. Dispatches the encoded call data from a given origin
+
+Therefore, XCM execution in the target chain consists of 3 to 4 XCM instructions, depending on the extrinsic being used. This section covers how XCM fees are estimated for each of the scenarios described above, as it is handled very differently.
+
+### Transact Through Derivative Fees
```
Comment (eshaben) [inline builders/xcm/fees.md:315]:
```suggestion
### Transact Through Derivative Fees {: #transact-through-derivative-fees }
```

File: builders/xcm/fees.md [hunk 13]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+Consider the following scenario: Alice wants to remotely transact in Polkadot from Moonbeam using the transact through sovereign extrinsic (she already has an index registered to her account). To estimate how many XC-20 tokens will be burned from Alice's account, you need to check the transact information specific to the relay chain. To do so, head to the chain state page of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/chainstate){target=_blank} and set the following options:
+
+1. Choose the **xcmTransactor** pallet
+2. Choose the **transactInfoWithWeightLimit** method
+3. Set the multilocation for the destination chain from which you want to query the transact information. For this example, you can set `parents` to `1` and `interior` to `here`
+4. Click on `+`
```
Comment (eshaben) [inline builders/xcm/fees.md:326]:
```suggestion
4. Click on **+**
```

File: builders/xcm/fees.md [hunk 14]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+PICTURE_HERE
+
+From the response, you can see that the `transactExtraWeight` is `{{ networks.polkadot.xcm_message.transact.weight }}`. This is the weight needed to execute the 3 XCM instructions for this remote call in that specific destination chain. Next, you need to find the `UnitsPerSecond` for that particular chain. In the same [Polkadot.js Apps page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/chainstate){target=_blank}, set the following options:
+
+1. Choose the **xcmTransactor** pallet
+2. Choose the **destinationAssetFeePerSecond** method
+3. Set the multilocation for the destination chain from which you want to query the transact information. For this example, you can set `parents` to `1` and `interior` to `here`
+4. Click on `+`
```
Comment (eshaben) [inline builders/xcm/fees.md:335]:
```suggestion
4. Click on **+**
```

File: builders/xcm/fees.md [hunk 15]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+PICTURE_HERE
+
+Note that this `UnitsPerSecond` is related to the cost estimated in the [Relay Chain XCM Fee Calculation](/builders/xcm/fees/#polkadot){target=_blank} section, or to the one shown in the [Units per weight](/builders/xcm/fees/#moonbeam-reserve-assets){target=_blank} section if the target is another parachain. As before, calculating the associated XCM execution fee is as simple as multiplying the `transactExtraWeight` times the `UnitsPerSecond`:
```
Comment (eshaben) [inline builders/xcm/fees.md:339]:
```suggestion
Note that this `UnitsPerSecond` is related to the cost estimated in the [Relay Chain XCM Fee Calculation](#polkadot) section, or to the [Wei per weight](#moonbeam-reserve-assets) if the target is another parachain. As before, calculating the associated XCM execution fee is as simple as multiplying the `transactExtraWeight` times the `UnitsPerSecond`:
```

File: builders/xcm/fees.md [hunk 16]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+Consider the following scenario: Alice wants to remotely transact in another chain (Parachain ID 888, in the Moonbase Alpha relay chain ecosystem) from Moonbase Alpha using the transact through signed extrinsic. To estimate the amount of tokens Alice's multilocation-derivative account will need to have to execute the remote call, you need to check the transact information specific to the destination chain. To do so, head to the chain state page of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/chainstate){target=_blank} and set the following options:
+
+
+1. Choose the **xcmTransactor** pallet
+2. Choose the **transactInfoWithWeightLimit** method
+3. Set the multilocation for the destination chain from which you want to query the transact information. For this example, you can set `parents` to `1` and `interior` to `X1: {"Parachain": 888}`
+4. Click on `+`
```
Comment (eshaben) [inline builders/xcm/fees.md:367]:
```suggestion
4. Click on **+**
```

File: builders/xcm/fees.md [hunk 17]
```diff
@@ -261,18 +277,118 @@ ExecutionTime = ({{ networks.moonbeam.xcm.instructions.weight_units.numbers_only
...
+
+PICTURE_HERE
+
+From the response, you can see that the `transactExtraWeightSigned` is `{{ networks.moonbase_beta.xcm_message.transact.weight }}`. This is the weight needed to execute the 4 XCM instructions for this remote call in that specific destination chain. Next, you need to find how much the destination chain charges per weight of XCM execution. Normally, you would look into the `UnitsPerSecond` for that particular chain. But in this scenario, no XC-20 tokens are burned. Therefore, `UnitsPerSecond` can be use for reference, but do not ensure that the amount of tokens estimated are correct. To get the `UnitsPerSecond` as a reference value, in the same [Polkadot.js Apps page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/chainstate){target=_blank}, set the following options:
+
+1. Choose the **xcmTransactor** pallet
+2. Choose the **destinationAssetFeePerSecond** method
+3. Set the multilocation for the destination chain from which you want to query the transact information. For this example, you can set `parents` to `1` and `interior` to `X2: [{"Parachain": 888},{"PalletInstance":3}]`
+4. Click on `+`
```
Comment (eshaben) [inline builders/xcm/fees.md:376]:
```suggestion
4. Click on **+**
```

File: builders/xcm/xcm-transactor.md [hunk 18]
```diff
@@ -13,7 +13,17 @@ XCM messages are comprised of a [series of instructions](/builders/xcm/overview/
...
+
+The two main extrinsics of the pallet are transacting through a sovereign derivative account or a derivative account calculated from a given multilocation. Each extrinsic is named accordingly.
+
+The relevant [XCM instructions](/builders/xcm/overview/#xcm-instructions) to perform remote execution through XCM are, but not limited to:
+
+ - [`WithdrawAsset`](https://github.com/paritytech/xcm-format#withdrawasset){target=_blank} - gets executed in the target chain. Removes assets and places them into the holding register
+ - [`BuyExecution`](https://github.com/paritytech/xcm-format#buyexecution){target=_blank} - gets executed in the target chain. Takes the assets from holding to pay for execution fees. The fees to pay are determined by the target chain
+ - [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} - gets executed in the target chain. Dispatches the encoded call data from a given origin
+
+When the XCM message built by the XCM-transactor pallet is executed, fees must be paid. All the relevant information can be found in the [XCM-transactor fees section](/builders/xcm/fees/#xcm-transactor-fees) of the [XCM fees](/builders/xcm/fees/) page.
```
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:26]:
```suggestion
When the XCM message built by the XCM-transactor pallet is executed, fees must be paid. All the relevant information can be found in the [XCM-transactor fees section](/builders/xcm/fees/#xcm-transactor-fees){target=_blank} of the [XCM fees](/builders/xcm/fees/){target=_blank} page.
```

File: builders/xcm/xcm-transactor.md [hunk 19]
```diff
@@ -139,6 +155,56 @@ To fetch a list of all registered addresses allowed to operate through the Moonb
...
+
+!!! note
+    You need to ensure that the call you are going to execute remotely is allowed in the destination chain!
+
+### Checking Prerequisites {: #xcmtransactor-signed-check-prerequisites}
+
+To be able to send the extrinsics in Polkadot.js Apps, you need to have:
+
+ - An [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts){target=_blank} in the origin chain with [funds](/builders/get-started/networks/moonbase/#get-tokens){target=_blank}
+ - Funds in the multilocation-derivative account on the target chain. You can calculate this address by using this [script](https://github.com/albertov19/xcmTools/blob/main/calculateMultilocationDerivative.ts){target=_blank}
```
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:171]:
```suggestion
 - Funds in the multilocation-derivative account on the target chain. You can calculate this address by using the [`calculateMultilocationDerivative.ts` script](https://github.com/albertov19/xcmTools/blob/main/calculateMultilocationDerivative.ts){target=_blank}
```

File: .snippets/text/xcm/general-xcm-definitions2.md [hunk 20]
```diff
@@ -1,3 +1,3 @@
...
+ - **Sovereign account** —  an accounts each chain in the ecosystem has, one for the relay chain and the other for other parachains. It is calculated as the `blake2` hash of a text (`para` for the sovereign account in the relay chain, and `sibl` for the sovereign account in other parachains), truncating the hash to the correct length. The account is owned by root and can only be used through SUDO (if available) or democracy (technical committee or referenda). The sovereign account typically signs XCM messages in other chains in the ecosystem
```
[the remark this answers is not in the document]
Comment (albertov19) [reply .snippets/text/xcm/general-xcm-definitions2.md:1]: Changed the definition

File: builders/xcm/fees.md (hunk 4 above)
Comment (albertov19) [reply builders/xcm/fees.md:164]: nice catch!

File: builders/xcm/fees.md (hunk 5 above)
Comment (albertov19) [reply builders/xcm/fees.md:201]: Yeah sure, you are more aware of the rules than me 😃

File: builders/xcm/fees.md (hunk 6 above)
Comment (albertov19) [reply builders/xcm/fees.md:198]: Agree

File: builders/xcm/fees.md (hunk 8 above)
Comment (albertov19) [reply builders/xcm/fees.md:107]: True, I've set `COST` to `Cost`

File: builders/xcm/fees.md (hunk 11 above)
Comment (albertov19) [reply builders/xcm/fees.md:307]: Yes, what I meant here is that the first instruction is always something to handle tokens in the origin chain. They differ depending if it is a Reserve asset or not. Any suggestions?

Comment (eshaben) [reply builders/xcm/fees.md:312]: Maybe start with just that... "The first instruction handles tokens in the origin chain. This can be..."

Comment (albertov19) [reply builders/xcm/fees.md:312]: Thanks, I like that

**Review by eshaben — APPROVED** [review]
thanks for fixing all my grammar errors! 😅 

just made a few minor changes but LGTM! ✨ 🚀 🚢

**Pull request merged**